pkg/testgridanalysis/testidentification/test_identification.go

```diff
@@ -66,25 +68,47 @@ var customJobInstallNames = sets.NewString(
 	"vsphere-upi-upi-install-vsphere",
 )
 
+const installStepName = "cluster install should succeed"
+
+var (
+	// The following two metrics are temporary counters helping us to decide when to remove old style identification.
+	totalJobRunWithOldInstallIdentificationMetric = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "sippy_total_runs_with_old_install_identification",
+		Help: "Total job runs with old style install identification",
+	})
+	totalJobRunWithNewInstallIdentificationMetric = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "sippy_total_runs_with_new_install_identification",
+		Help: "Total job runs with new style install identification",
+	})
+)
+
 // TODO We should instead try to detect whether we fail in a pre-step to determine whether install succeeded
 // Install steps have different names in different jobs. This is heavily dependent on the actual UPI jobs, but they turn out to be different.
 // When this needs updating,  it shows up as installs timing out in weird numbers
 func IsInstallStepEquivalent(testName string) bool {
```

Comment:
Where do we compute the infrastructure failures?  I think we should add the install failures on infra to the infra failed percentage.

We should also add the bootstrapping failure and infrastructure failure to the list of interesting install tests on the install details page.  I think that vsphere is failing bootstrapping a lot (as a for instance).